apps/journeys-admin/src/components/SingleJourney/CardOverview/CardOverview.tsx

```diff
@@ -1,26 +1,74 @@
-import { ReactElement } from 'react'
-import { Button, Box, Typography } from '@mui/material'
+import { ReactElement, useState, useEffect } from 'react'
+import { Button, Typography, useTheme } from '@mui/material'
 import EditIcon from '@mui/icons-material/Edit'
 
+import { BlockFields_StepBlock as StepBlock } from '../../../../__generated__/BlockFields'
+import { CardPreview } from '../../CardPreview'
+import { TreeBlock } from '@core/journeys/ui'
+
 export interface CardOverviewProps {
   slug: string
+  blocks?: Array<TreeBlock<StepBlock>>
```

Comment:
```suggestion
  blocks: Array<TreeBlock<StepBlock>>
```
Maybe better to always pass in blocks, even if it's an empty array - because the main purpose of this component is to display blocks..